vignettes/ggPMX-ggplot.Rmd

```diff
@@ -0,0 +1,81 @@
+---
+title: "How to use ggplot2 syntax with ggPMX"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to use ggplot syntax with ggPMX}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
```

Comment:
This chunk is consistent with the `tidyverse` methods of creating vignettes but not consistent internally.  Either change the rest to have the `#>`  / `collapse` options.  Also other vignettes have `knitr::opts_chunk$set(out.width = "100%", warning = FALSE, message = FALSE)` which is inconsistent here.

Reply:
Yes, will use one  knitr::opts_chunk$set call instead of set options in each chunk.